contribs/minibus/src/main/java/org/matsim/contrib/minibus/hook/OperatorCostCollectorHandler.java

```diff
@@ -165,4 +157,33 @@ public void notifyAfterMobsim(AfterMobsimEvent event) {
 			}
 		}
 	}
+
+	public Map<Id<Vehicle>, OperatorCostContainer> getVehicleIdToOperatorCostContainerMap(){
+		return Collections.unmodifiableMap( this.vehId2OperatorCostContainer );
+	}
+
+	@Override
+	public void handleEvent(PersonMoneyEvent event) {
+		// yyyy what comes below is not so beautiful since it depends on conventions: If some string contains something of some other string,
+		// then ... It looks like this was the original design of the minibus contrib, so in principle it should be operated out at some
+		// point, but probably not now.  kai, jul'21
+
+		if(event.getPersonId().toString().contains(this.pIdentifier)){
+			/* It is a minibus driver. Now find the correct vehicle this person is driving. */
+			for(Id<Vehicle> vehicleId : vehId2OperatorCostContainer.keySet()){
+				/* The driver Id contains the vehicle Id. For example, if the
+				vehicle Id is para_1_2 then you will find that the driver's Id
+				is something like pt_para_1_2_para_. I do not know where this
+				is set/created, but it may change in the future (JWJ '21) */
```

Comment:
We can find out, but it does not seem useful to invest much time into that given Kai's comment above. Looping over all vehicles does not seem very efficient, but for a temporary solution it should be ok unless you report performance issues having a serious impact on you, @JWJoubert .